components/organisms/Header/Header.tsx

```diff
@@ -3,15 +3,19 @@ import { HeaderStyle } from './style'
 import { Border } from '@/components/atoms/Border'
 import { Menu } from '@/components/molecules/Menu'
 import { menuItems as items } from '@/constants/menu'
+import { Container } from '@/components/atoms/Container'
 
 type PropsType = {
+  // eslint-disable-next-line no-unused-vars
   onClickMenuItem: (item: string) => void
 }
 
 const Header: React.FC<PropsType> = ({ onClickMenuItem }) => {
   return (
     <header css={HeaderStyle}>
-      <Menu onClickMenuItem={onClickMenuItem} items={items} />
+      <Container>
+        <Menu onClickMenuItem={onClickMenuItem} items={items} />
+      </Container>
       <Border />
     </header>
   )
```

Comment:
This is necessary because the goal is to ignore warnings that are already noticed and are not important.
If there is a way to avoid this warning when passing arguments to a function of type void, I would adopt it.

Reply:
Seems it not works as expected.
```
9:21  error  '_item' is defined but never used  no-unused-vars
```

Reply:
Thanks.
Updated the ESLint configuration as follow.
```
{
  "extends": ["eslint:recommended", "next", "next/core-web-vitals", "prettier"],
  "rules": {
    "no-unused-vars": [
      "warn",
      {
        "argsIgnorePattern": "^_",
        "varsIgnorePattern": "^_",
        "caughtErrorsIgnorePattern": "^_",
        "destructuredArrayIgnorePattern": "^_"
      }
    ]
  }
}

```